tags.go

```diff
@@ -0,0 +1,89 @@
+package brokerTags
+
+import (
+	"time"
+)
+
+type TagManager struct {
+	broker          string
+	cfClientWrapper CFClientWrapper
+}
+
+func NewTagManager() (*TagManager, error) {
+	cfClientWrapper, err := NewCFClientWrapper()
+	if err != nil {
+		return nil, err
+	}
+	return &TagManager{
+		cfClientWrapper: cfClientWrapper,
+	}, nil
+}
+
+func (t *TagManager) GenerateTags(
+	action string,
```

Comment:
Most of these are self-documenting, but `action` might benefit from a doc comment